src/map/grid.rs

```diff
@@ -15,3 +15,18 @@ pub struct Grid {
     pub rows: usize,
     pub tiles: Vec<Arc<Tile>>,
 }
+
+impl Grid {
+    pub fn passable(&self, tilepos: (isize, isize)) -> bool {
+        if tilepos.0 < 0 || tilepos.1 < 0 {
```

Comment:
hmm i know is this was ASSembly we could just to an unsigned compare to get whether we're in the map bounds is there a way to do this in ruust

Reply:
that said considering that with proper map gen we shouldn't even need to concern ourselves with going out of bounds, as there should always be some wall bordering which we can actually draw, also how necessary is this comparison to begin with?

Reply:
like i dont think we will ever want the map to directly touch the screen edge

Reply:
well actually we will prolly add destructible enviornments so nvm disregard that point

Reply:
Ok disregarded.

Reply:
no i just meant the point about whether they were necessary my question about doing an unsigned compare stands